LSP-json.sublime-settings

```diff
@@ -21,6 +21,8 @@
 			],
 		}
 	],
+	"auto_complete_selector": "- comment - punctuation.separator", // don't auto-complete in comments and after typing a "," or ":"
+	"allow_completion_triggers_from_server": false,
```

Comment:
I haven't pushed this setting yet in the LSP pull request :)

Reply:
And now it is https://github.com/sublimelsp/LSP/pull/1408/commits/2fd4f8e309f36e53d9103870c6c8fc144373fece